ios/dcframework/generatedTests/DINTDaggerTestComponent.h

```diff
@@ -0,0 +1,71 @@
+//
+//  Generated by the J2ObjC translator.  DO NOT EDIT!
```

Comment:
Should generated files be included in source control? Should probably add to gitignore?